internal/rooms/rooms.go

```diff
@@ -1576,151 +1576,86 @@ func (r *Room) FindContainerByName(containerNameSearch string) string {
 	return closeMatch
 }
 
+// FindNoun resolves a noun or alias within the room's noun map.
 func (r *Room) FindNoun(noun string) (foundNoun string, nounDescription string) {
-
 	if len(r.Nouns) == 0 {
 		return ``, ``
 	}
 
-	roomNouns := map[string]string{}
-
-	for originalNoun, originalDesc := range r.Nouns {
-		roomNouns[originalNoun] = originalDesc
-
-		if strings.Contains(originalNoun, ` `) {
-			for _, n := range strings.Split(originalNoun, ` `) {
-
-				if _, ok := r.Nouns[n]; ok {
-					continue
-				}
-
-				if _, ok := roomNouns[n]; ok {
-					continue
+	// Build flat map of noun -> desc, and add single-word aliases for multi-word nouns
+	roomNouns := make(map[string]string, len(r.Nouns))
+	for key, desc := range r.Nouns {
+		roomNouns[key] = desc
+		if strings.Contains(key, ` `) {
+			for word := range strings.SplitSeq(key, " ") {
+				if _, exists := r.Nouns[word]; !exists && roomNouns[word] == "" {
```

Comment:
If `!exists` is true, the value of `roomNouns[word]` will always be the default value of string, AKA `""`. The check for the value should be unnecessary.